### PR TITLE
Create feature list

### PR DESCRIPTION
**Pull request by VishakhaVaidya**

### CODE REVIEW

File: CEP_8.x/FeatureList8.0 [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+* CEF and nodejs upgrade [Chromium version#57.0.2987.74 with nodeJS version#7.7.4]
+* IME Support on top of latest CEF
+* Windows High DPI Support – 125% scaling supported
+* Bug Fixes
+  * Crashes
+    * CEP extensions crashing in background
+    * Application hang in PlugPlugOWL (CEP) MonitorThreadProc while application quit
+    * MAC Only: CEPHtmlEngine crashes in background while using Illustrator.
+    * AI crashes during PlugPlugOWL is being unloaded.
+    * Crash on Fist Launch of illustrator after ESD install
```
Comment (venpa) [inline CEP_8.x/FeatureList8.0]: Correct the spelling.

File: CEP_8.x/FeatureList8.0 [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+==============
+* CEF and nodejs upgrade [Chromium version#57.0.2987.74 with nodeJS version#7.7.4]
+* IME Support on top of latest CEF
+* Windows High DPI Support – 125% scaling supported
+* Bug Fixes
+  * Crashes
+    * CEP extensions crashing in background
+    * Application hang in PlugPlugOWL (CEP) MonitorThreadProc while application quit
+    * MAC Only: CEPHtmlEngine crashes in background while using Illustrator.
+    * AI crashes during PlugPlugOWL is being unloaded.
```
Comment (venpa) [inline CEP_8.x/FeatureList8.0]: Use AI or Illustrator every where in the document.

Comment (VishakhaVaidya) [reply CEP_8.x/FeatureList8.0]: Done

File: CEP_8.x/FeatureList8.0 (hunk 1 above)
Comment (VishakhaVaidya) [reply CEP_8.x/FeatureList8.0]: Done

**Pull request merged**